js/src/product-feed/product-statistics/create-campaign-notice/index.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Notice } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import AddPaidCampaignButton from '.~/components/paid-ads/add-paid-campaign-button';
+import useMCProductStatistics from '.~/hooks/useMCProductStatistics';
+import useAdsCampaigns from '.~/hooks/useAdsCampaigns';
+import './index.scss';
+
+const CreateCampaignNotice = () => {
+	const { hasFinishedResolution, data: products } = useMCProductStatistics();
+	const { loaded: campaignsLoaded, data: campaigns } = useAdsCampaigns();
+
+	const isLoading =
+		! hasFinishedResolution || products?.loading || ! campaignsLoaded;
+
+	if (
+		isLoading ||
+		! products ||
+		products?.statistics?.active === 0 ||
+		! campaigns ||
+		campaigns?.length > 0
+	) {
+		return null;
+	}
+
+	return (
+		<Notice isDismissible={ false } className="gla-ads-inline-notice">
```

Comment:
Sure, it could be. The reason it was done this way is because it was described in the designs as a component that existed in the pattern library called an "inline notice", but there did not yet seem to be an implementation of such a component in this plugin. Semantically, implementing this as a new `Notice` type makes sense and the current implementation does match the design intent.

Reply:
[165543c](https://github.com/woocommerce/google-listings-and-ads/pull/2641/commits/165543cc1f6e87d45b75660203e1c081476335f0) swaps this to a `Flex` which also allows it to be rendered in the `CardFooter`.